src/persist-client/src/internal/state.rs

```diff
@@ -248,7 +248,7 @@ where
             // instead of getting it from Register or (2) if a lease expired.
             // (1) is a gross mis-use and (2) isn't implemented yet, so it feels
             // okay to leave this for followup work.
-            .expect("TODO: Implement automatic lease renewals")
+            .unwrap_or_else(|| panic!("ReaderId({}) does not exist", id))
```

Comment:
super nit: can we make this message something like "reader's lease was expired through inactivity". once we get the leasing issues worked out, anything that hits this panic will be an error in persist usage and I want to make it sound a bit more like that